include/pmacc/kernel/atomic.hpp

```diff
@@ -67,8 +67,8 @@ namespace pmacc
                 }
             };
 
-#if(!defined(__CUDA__) && ALPAKA_ACC_GPU_HIP_ENABLED == 1 && (HIP_VERSION_MAJOR * 100 + HIP_VERSION_MINOR) < 401)
-            /** HIP backend specialization for atomic add
+#if(ALPAKA_ACC_GPU_HIP_ENABLED == 1 && (HIP_VERSION_MAJOR * 100 + HIP_VERSION_MINOR) < 401)
```

Comment:
What should happen when `HIP_VERSION_MAJOR * 100 + HIP_VERSION_MINOR` equals 401 or 403 ?

Reply:
if it is >= 403 it will us ethe new workaround. For version <401 (not 401) it is using the not specialized version, the alpaka implementation.

Reply:
Yes, i can see that. I am wondering what's happening if it is between those two checks, for 401 and 402. Or do they not exist?

Reply:
Discussed offline: for these versions nothing has to be done.